DSCResources/xSmaVariable/xSmaVariable.schema.mof

```diff
@@ -0,0 +1,11 @@
+
+[ClassVersion("1.0.0.0"), FriendlyName("SmaVariable")]
+class xSmaVariable : OMI_BaseResource
+{
+    [Key] String Name;
+    [Required] String value;
+    [Write] String Description;
+    [Read] Boolean Set;
+    [Required] String WebServiceEndpoint;

```

Comment:
Port gets complicated.  Since you want it to be optional.  You know how to IM me if you want to discuss it.